src/components/graph/MetaGraph.ts

```diff
@@ -185,19 +188,22 @@ export class MetaGraph {
 
     updateGraph(metaNodeModel: MetaNodeModel, cursorX: number, cursorY: number) {
         // update the graph for right parent children relationship
+        let pathUpdated = false;
```

Comment:
@ddelpiano what about exit the if then else (with a `return`) and remove this variable